README.md

```diff
@@ -16,48 +22,26 @@ drand was first developed within the [DEDIS organization](
 https://github.com/dedis), and as of December 2019,
 is now under the drand organization.
 
-### Disclaimer
-⚠️ **This software is considered experimental and has NOT received a third-party
-audit yet. Therefore, DO NOT USE it in production or for anything security
-critical at this point.**
-
 ## Table of Contents
 
 <!-- START doctoc generated TOC please keep comment here to allow auto update -->
 <!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
 
+
 - [Goal and Overview](#goal-and-overview)
   - [Public Randomness](#public-randomness)
   - [Private Randomness](#private-randomness)
-- [Local demo](#local-demo)
-- [Installation](#installation)
-  - [Official release](#official-release)
-  - [Manual installation](#manual-installation)
+- [Try it for yourself](#try-it-for-yourself)
+  - [Installation](#installation)
```

Comment:
"Installation" and "Usage" is now "under" "Try it for yourself" - is that intended? I would keep "Install" and "Usage" top level (and would keep conformance with standard README).